module/lua/Makefile.in

```diff
@@ -0,0 +1,69 @@
+src = @abs_top_srcdir@/module/lua
+obj = @abs_builddir@
+
+MODULE := zlua
+
+TARGET_ASM_DIR = @TARGET_ASM_DIR@
```

Comment:
`TARGET_ASM_DIR` is defined in `always-arch.m4` with the `ZFS_AC_CONFIG_ALWAYS_ARCH` macro.  However, it is currently only defined for `i386|x86_64` it will need to be extends for the other architectures.

Reply:
Will try and fix this inside lua section.